### PR TITLE
Label scenario

### LINKED ISSUE #230: In plot_techmix*() we don't konw which scenario we are plotting.
Do we need a title showing which scenario (e.g. "sds") we are plotting?

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #230 

In this PR I change the labelling functions so that the scenario name is part of `qplot_techmix()` label.

### CODE REVIEW

File: R/r2dii.plot-package.R [hunk 1]
```diff
@@ -6,5 +6,6 @@
 ## usethis namespace: start
 #' @importFrom dplyr n
 #' @importFrom scales percent
+#' @importFrom stringr str_count
```
Comment (jdhoffa) [inline R/r2dii.plot-package.R:9]: I would tend to prefer calling `stringr::str_count` so it's explicit where the `stringr` dependency is?

Comment (jdhoffa) [reply R/r2dii.plot-package.R:9]: Ah it looks like you do anyways. So you can probably remove this line

Comment (MonikaFu) [reply R/r2dii.plot-package.R:9]: I think that if I remove this line then the checks will not pass?

**Review by jdhoffa — APPROVED** [review]
Super minor comment, but lgtm

**Pull request merged**